docs/sql/statements/grant.rst

```diff
@@ -18,17 +19,24 @@ Synopsis
 
   GRANT { { DQL | DML | DDL | AL [,...] } | ALL [ PRIVILEGES ] }
   [ON {SCHEMA | TABLE | VIEW} identifier [, ...]]
-  TO user_name [, ...];
+  TO name [, ...]
+
+  GRANT role_name_to_grant [, ...] TO name [, ...]
 
 Description
 ===========
 
-``GRANT`` is a management statement to grant one or many privileges
-on the whole cluster or on a specific object to one or many existing users.
+``GRANT`` is a management statement which comes in two flavours.
 
+The first one is used to grant one or many privileges on the whole cluster or on
+a specific object to one or many existing users or roles.
 ``ON {SCHEMA | TABLE | VIEW}`` is optional, if not specified the privilege will
 be granted on the ``CLUSTER`` level.
 
+With the second one, ``GRANT`` can be used to grant one or more roles to one or
+many others or roles. Thus, the users (or roles) inherit the privileges of the
+roles which they are granted.
```

Comment:
Since |inherit" is mentioned - let' link it to `:ref:granted`<roles_inheritance`>`?